keyboards/genone/eclipse_65/readme.md

```diff
@@ -0,0 +1,17 @@
+# GEN ONE Eclipse_65
+
+Eclipse_65 is a keyboard PCB supporting 65% layout. [More info at GEN ONE MERCH](https://genonemerch.com)
```

Comment:
This link seems to be dead.

Reply:
I tried remaking the file using MSYS2 and received an error occurring with the keymap.c file. It says it has an error with the change from LAYOUT_ALL to LAYOUT_65_ANSI.

Reply:
# make genone/eclipse_65:default
QMK Firmware 0.9.36
WARNING:
 Can not run bin/qmk! This tool will be required when the develop branch is merged on 2020 Aug 29.

 Please run util/qmk_install.sh to install all the dependencies QMK requires.

Making genone/eclipse_65 with keymap default

avr-gcc.exe (GCC) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text    data     bss     dec     hex filename
      0   14414       0   14414    384e .build/genone_eclipse_65_default.hex

Compiling: keyboards/genone/eclipse_65/keymaps/default/keymap.c                                    keyboards/genone/eclipse_65/keymaps/default/keymap.c:22:8: error: implicit declaration of function 'LAYOUT_65_ansi'; did you mean 'LAYOUT_all'? [-Werror=implicit-function-declaration]
  [0] = LAYOUT_65_ansi(
        ^~~~~~~~~~~~~~
        LAYOUT_all
keyboards/genone/eclipse_65/keymaps/default/keymap.c:22:8: error: initializer element is not constant
keyboards/genone/eclipse_65/keymaps/default/keymap.c:22:8: note: (near initialization for 'keymaps[0][0][0]')
keyboards/genone/eclipse_65/keymaps/default/keymap.c:30:8: error: initializer element is not constant
  [1] = LAYOUT_65_ansi(
        ^~~~~~~~~~~~~~
keyboards/genone/eclipse_65/keymaps/default/keymap.c:30:8: note: (near initialization for 'keymaps[1][0][0]')
cc1.exe: all warnings being treated as errors
 [ERRORS]
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:386: .build/obj_genone_eclipse_65_default/keyboards/genone/eclipse_65/keymaps/default/keymap.o] Error 1
Make finished with errors
make: *** [Makefile:584: genone/eclipse_65:default] Error 1

Reply:
Well, CI is able to build it just fine so I think you are probably missing some of those changes locally.

Reply:
Thanks for the response. 
I changed all files that you requested me to change. What is missing from getting your approval?

Reply:
You accepted all of my suggestions above, but did you then pull those changes down to your local clone from GitHub?

Other than the dead link here, everything looks good.

Reply:
I accepted and committed all suggestions you posted above. I manually removed main.c file and manually corrected the keymap.c file. When I open the files in my fork, they are all changed with you suggestions. 

Is the dead link a problem? how would I go about fixing that?

Reply:
I get this when I go to that page:
![image](https://user-images.githubusercontent.com/4781841/97084569-bfc71200-1663-11eb-9f5d-1d545a2e826f.png)

Reply:
Ok, that's what the link is. We purchased the domain name and are currently working on the website. We are going to have it running next week.

Reply:
How do I get ZVECR for another review for approval?